recipes/ssh2-python/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set name = "ssh2-python" %}
+{% set version = "0.5.1" %}
```

Comment:
This PR was made before 0.5.3 (I am the author). I'd rather work on getting this merged first and follow up with an update once it is.